src/barkhaPJun20/FunctionsOfPrimeNum.java

```diff
@@ -0,0 +1,66 @@
+package barkhaPJun20;
+
+public class FunctionsOfPrimeNum {
+	int sum=0;
+	int count=0;
+	
+	void primeOrNot(int num) {
+		boolean flag=true;    
+		for (int index=2; index<=num/2; index++) {
+			if (num%index==0) {
+				flag=false;  // if flag false then not Prime Num
+				break;
+			}	
+		}	
+		if (flag==true)
+			System.out.println("Number "+num+" is Prime Number.");
+		else
+			System.out.println("Number "+num+" is Not Prime Number.");
+	}
+	void printCountOfPrimeNum() {
+		//int count=0;
+		
+		for (int number=2; number<=100; number++) {
+			boolean flag=true; 
+			for (int index=2; index<=number/2; index++) {
+				if (number%index==0) {
+					flag=false;
+					break;
+				}
+			}if (flag==true)	
+				count++;	
+				sum=sum+number;
+			}		
+		System.out.println("Total count of Prime numbers in the range 2-100 are: "+count);		
+	}
+	void printSumOfPrimeNum() {	
+	
+		System.out.println("Total sum of Prime numbers in the range 2-100 are: "+sum);
+	}
+	void printAvgOfPrimeNum() {
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
verifyNumberIsPrimeOrNot2(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count);